src/flb_engine.c

```diff
@@ -69,12 +69,17 @@ extern struct flb_aws_error_reporter *error_reporter;
 
 #include <ctraces/ctr_version.h>
 
+static pthread_once_t local_thread_engine_evl_init = PTHREAD_ONCE_INIT;
 FLB_TLS_DEFINE(struct mk_event_loop, flb_engine_evl);
 
+void flb_engine_evl_init_private()
```

Comment:
Make this static please.

Reply:
Oh, I'd overlooked that. Thanks for pointing out.